suite/runsuite_wrapper.pl

```diff
@@ -42,11 +42,13 @@
 use File::Basename;
 my $mydir = dirname(abs_path($0));
 my $is_CI = 0;
+my $is_AArchxx = 0;
 
 # Forward args to runsuite.cmake:
 my $args = '';
 for (my $i = 0; $i <= $#ARGV; $i++) {
     $is_CI = 1 if ($ARGV[$i] eq 'travis');
+    $is_AArchxx = 1 if ($ARGV[$i] eq 'aarchxx');
```

Comment:
I don't see anyone passing "aarchxx" so I have to assume you will pass it from Jenkins, instead of having this script run uname or use Config or sthg.

But, runsuite.cmake has no code handling such an arg, so should it really be passed to runsuite.cmake?